src/OpenFeature.DependencyInjection/OpenFeatureServiceCollectionExtensions.cs

```diff
@@ -24,7 +24,7 @@ public static IServiceCollection AddOpenFeature(this IServiceCollection services
         Guard.ThrowIfNull(configure);
 
         // Register core OpenFeature services as singletons.
-        services.TryAddSingleton(Api.Instance);
+        services.TryAddSingleton(new Api());
```

Comment:
I think having managing the singleton via the DI framework makes sense, but doing this still results in 2 (we still have another call to `new Api()` Api.cs).

I guess there's no easy way around this? Is there anything we can do to avoid both instances?

Reply:
Are you talking about the `static` constructor?
That one is just a technique to lazy load the singleton. My concern is more if people start to mismatch calls using the`IFeatureClient` via DI and `API.Instance`, which I don't know how we could prevent it.

Reply:
I'm saying that we now have 2 instances of the singleton (unless I'm really misunderstanding something) one managed by the SDK, instantiated [here on line 35](https://github.com/open-feature/dotnet-sdk/pull/534/files#diff-dc150fbd3b7be797470374ee7e38c7ab8a31eb9b6b7a52345e05114c2d32a15eR29), and another in the DI framework.